config/pulse.php

```diff
@@ -79,6 +79,8 @@
     'ingest' => [
         'driver' => env('PULSE_INGEST_DRIVER', 'storage'),
 
+        'buffer' => env('PULSE_INGEST_BUFFER', 5_000),
+
```

Comment:
This is totally arbitrary. I don't know what a good "default" for this would even be.

`5_000` felt good at the time 🤷‍♂️